proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -354,3 +357,12 @@ message PayloadSCIMTokenCreated {
 }
 
 message PayloadSCIMTokenDeleted {}
+
+message PayloadDeleteRepositoryDraftCommit {
+  // owner_name is the name of the owner of the repository from which the draft will be deleted.
+  string owner_name = 1;
+  // repository_name is the name of the repository from which the draft will be deleted.
+  string repository_name = 2;
```

Comment:
Should we add `owner_id` and `repository_id` here to be similar to the push commit message?